testing/test_pytester.py

```diff
@@ -413,7 +413,7 @@ def test_testdir_run_with_timeout(testdir):
     testfile = testdir.makepyfile("def test_no_timeout(): pass")
 
     start = time.time()
-    result = testdir.runpytest_subprocess(testfile, timeout=10)
+    result = testdir.runpytest_subprocess(testfile, timeout=120)
```

Comment:
@altendky 
Wouldn't that still fail with `assert duration < 5` below?

Reply:
Unfortunately the build failure is gone, likely because it was restarted?!  (would be good to copy'n'paste those in the future).  (it was for py27-pluggymaster)

Reply:
:[ Yes, I should have extended that as well.  Or, really, made one number calculated from the other to avoid this error in the future.

How should I proceed?  Push a new commit to the branch?  Start a new branch?

And :[ again.  I did double check that my link to the broken build was for a specific build.  I didn't realize that Travis was going to reuse that number and effectively break the link.  Lesson learned.  *sigh* Don't restart builds, trigger a new build (empty commit, close/reopen PR, something).